stories/Introduction.tsx

```diff
@@ -1,13 +1,18 @@
 import * as React from 'react';
 import { storiesOf } from '@storybook/react';
 import Markdown from 'wix-storybook-utils/Markdown';
+import { StoryCategory } from './storyHierarchy';
 const README = require('../README.md').default;
 const Usage = require('../docs/USAGE.md').default;
 const RTL = require('../docs/RTL.md').default;
 const Contribution = require('../docs/CONTRIBUTION.md').default;
+const ComponentGuideline = require('../docs/COMPONENT_GUIDELINES.md').default;
 
-storiesOf('Getting Started', module)
-  .add('Introduction', () => <Markdown source={README} />)
+storiesOf(StoryCategory.CONSUMING, module)
+  .add('Overview', () => <Markdown source={README} />)
   .add('Using the library', () => <Markdown source={Usage} />)
-  .add('Using RTL', () => <Markdown source={RTL} />)
-  .add('Contributing', () => <Markdown source={Contribution} />);
+  .add('RTL Support', () => <Markdown source={RTL} />);
```

Comment:
Maybe we should also align the title inside the file?

Reply:
👌